apertium/Makefile.am

```diff
@@ -290,7 +291,7 @@ apertium_postlatex_raw_SOURCES = apertium_postlatex_raw.cc
 apertium_postlatex_raw_LDADD= $(APERTIUM_LIBS) -lapertium$(GENERIC_MAJOR_VERSION) $(lib_LTLIBRARIES)
 
 apertium_tagger_SOURCES = apertium_tagger.cc
-apertium_tagger_LDADD = $(APERTIUM_LIBS) -lapertium$(GENERIC_MAJOR_VERSION) $(lib_LTLIBRARIES)
+APERTIUM_tagger_LDADD = $(APERTIUM_LIBS) -lapertium$(GENERIC_MAJOR_VERSION) $(lib_LTLIBRARIES)
```

Comment:
this change seems like an error?

Reply:
It must be an error. I did not notice the change.